### DISCUSSION TITLE
fluxconfig-agent not starting

### DISCUSSION THREAD

**Discussion by JoshDouglasOU** [fluxcd/flux2#3006] [General] [answered]
I am getting an error with fluxconfig agent that I haven't been able to solve.

✗ a container name must be specified for pod fluxconfig-agent-5d98b7c568-stt2f, choose one of: [fluxconfig-agent fluent-bit]

I also get this on the node
message: 'Readiness probe failed: HTTP probe failed with statuscode: 500'

**Comment by JoshDouglasOU** [comment]
i'm a little bit farther. It's aadpodidentity that is crashing the agent.  Pretty sure I need an exception but so far my exceptions do not look like they are working

**Comment by stefanprodan** [comment, marked as answer]
There is no such thing as a `fluxconfig-agent` in the CNCF Flux distribution. Please reach out to Azure support.
